polar-core/src/vm.rs

```diff
@@ -2853,13 +2929,9 @@ impl Runnable for PolarVirtualMachine {
             self.maybe_break(DebugEvent::Goal(goal.clone()))?;
         }
 
-        if self.log {
-            self.print("⇒ result");
-            if self.tracing {
-                for t in &self.trace {
-                    self.print(&format!("trace\n{}", t.draw(self)));
-                }
-            }
+        self.log_with(LogLevel::Trace, || &"⇒ result", &[]);
```

Comment:
This seems kind of duplicative in light of the new `Info`-level `RESULT: ...` log on 3019 below